contracts/gateway/UtilityToken.sol

```diff
@@ -180,6 +180,18 @@ contract UtilityToken is EIP20Token, Organized, UtilityTokenInterface {
         success_ = decreaseSupplyInternal(_amount);
     }
 
+    /**
+     * @notice Checks if an address is allowed to hold utility token.
+     *
+     * @dev In this case all the addresses are allowed to hold the utility
```

Comment:
dev. For standard ethereum all account addresses exist by default, so it returns true for all addresses.